src/main/java/ai/lucidtech/las/sdk/Credentials.java

```diff
@@ -47,7 +48,7 @@ public Credentials() {
      * @param credentialsPath Path to credentials file
      */
     public Credentials(String credentialsPath) {
-        this.readFromFile(credentialsPath);
+        this.setCredentials(credentialsPath);
```

Comment:
I don't think setting explicit credentials path should be overrided by ENV. I think the order in which credentials are looked up should only apply when constructing credentials without parameters. As long as we explicitly tell the program to use other credentials we should honor that request.